rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-10-17
```

Comment:
Isn't specifying a working nightly version safer? Things can break between versions

Reply:
On the nightly yes, breakages could happen, but upon further inspection, the only reason to use nightly is only used for a test in `grammatec`. I will push up a modification to this branch to move to stable. All targets are building and all tests passing